syntax.php

```diff
@@ -13,19 +13,9 @@
  * @author  Michael Klier <chi@chimeric.de>
  * @author  Mark C. Prins <mprins@users.sf.net>
  */
-if (!defined('DOKU_INC')) {
-    die();
-}
-
-if (!defined('DOKU_PLUGIN')) {
-    define('DOKU_PLUGIN', DOKU_INC.'lib/plugins/');
-}
-if (!defined('DW_LF')) {
-    define('DW_LF', "\n");
-}
 
-require_once(DOKU_PLUGIN.'syntax.php');
-require_once(DOKU_INC.'inc/parserutils.php');
+// must be run within Dokuwiki
+if (!defined('DOKU_INC')) die();
```

Comment:
Though i would agree with you, this line was taken from official syntax-template skeleton. So it should be fine to have it like all other plugins.